docs/user_guides/server-guide.md

```diff
@@ -84,6 +84,42 @@ Sample response:
 }]
 ```
 
+### Threads
+
+The Guardrails Server has basic support for storing the conversation threads. This is useful when you can only send the latest user message for a conversation rather than the entire history (e.g., from a third-party integration hook).
```

Comment:
```suggestion
The Guardrails Server has basic support for storing the conversation threads. This is useful as you can send only the latest user message(s) for a conversation rather than the entire history (e.g., from a third-party integration hook).
```